apps/jobs/views.py

```diff
@@ -234,32 +239,48 @@ def challenge_submission(request, challenge_id, challenge_phase_id):
             >= challenge_phase.max_concurrent_submissions_allowed
         ):
             message = "You have {} submissions that are being processed. \
-                       Please wait for them to finish and then try again."
+                Please wait for them to finish and then try again."
             response_data = {"error": message.format(submissions_in_progress)}
             return Response(
                 response_data, status=status.HTTP_406_NOT_ACCEPTABLE
             )
 
-        serializer = SubmissionSerializer(
-            data=request.data,
-            context={
-                "participant_team": participant_team,
-                "challenge_phase": challenge_phase,
-                "request": request,
-            },
-        )
-        if serializer.is_valid():
-            serializer.save()
-            response_data = serializer.data
-            submission = serializer.instance
-            # publish message in the queue
-            publish_submission_message(
-                challenge_id, challenge_phase_id, submission.id
+        if 'file_url' in request.data:
```

Comment:
Please use the condition from here: https://github.com/Cloud-CV/EvalAI/pull/1926/files#diff-42c752186f5f6b2ed96b92afa242a6f3R166

It will be shorter and crisp.